extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -72,6 +74,17 @@ export class BaseCarousel extends AMP.BaseElement {
     return true;
   }
 
+  hintControls() {
+    if (this.element.hasAttribute('controls')) {
```

Comment:
lets just cache this in a prop, since we also check for hasAttribute('controls') in buildCallback